source/gwork/include/Gwork/Skins/TexturedBase.h

```diff
@@ -1006,7 +1003,7 @@ namespace Gwk
                 Textures.Input.Slider.H.Normal.DrawCenter( GetRender(), control->GetRenderBounds() );
             }
 
-            void DrawTreeNode( Controls::Base* ctrl, bool bOpen, bool bSelected, int iLabelHeight, int iLabelWidth, int iHalfWay, int iLastBranch, bool bIsRoot )
+            virtual void DrawTreeNode( Controls::Base* ctrl, bool bOpen, bool bSelected, int iLabelHeight, int iLabelWidth, int iHalfWay, int iLastBranch, bool bIsRoot ) override
```

Comment:
While I'd be willing to remove the redundant virtuals, I'd prefer to do so as part of a larger overhaul; TextBox.h, for example, has about a full page worth that need to be removed.

I'd be willing to go through and get rid of all of them, but it may be better to do that in a separate pull request.